Toolchain/Patches/gcc/0001-Add-a-gcc-driver-for-SerenityOS.patch

```diff
@@ -99,7 +99,7 @@ index 0000000000000000000000000000000000000000..10c6f16fbe13eab36bab8f6896f8e2a6
 +/* Default arguments you want when running your
 +   x86_64-serenity-gcc toolchain */
 +#undef LIB_SPEC
-+#define LIB_SPEC "-lc" /* link against C standard library */
++#define LIB_SPEC "-lruntime -lc" /* link against C standard library and LibRuntime */
```

Comment:
I'm not sure if it's better to leave it this way or build a special libc with bundled LibRuntime for ports